packages/block-library/src/social-link/edit.js

```diff
@@ -27,13 +29,35 @@ import { keyboardReturn } from '@wordpress/icons';
  */
 import { getIconBySite, getNameBySite } from './social-list';
 
-const SocialLinkEdit = ( { attributes, setAttributes, isSelected } ) => {
+const SocialLinkEdit = ( {
+	attributes,
+	setAttributes,
+	isSelected,
+	context,
+	rgbIconColor,
```

Comment:
Could these just be `iconColor` and `backgroundColor` or does the `usesContext` mean they have to be named differently?

Reply:
Oh I see the naming clash now in edit.